v23.1/create-schedule-for-changefeed.md

```diff
@@ -0,0 +1,154 @@
+---
+title: CREATE SCHEDULE FOR CHANGEFEED
+summary: The CREATE SCHEDULE FOR CHANGEFEED statement creates a schedule for periodic changefeeds.
+toc: true
+docs_area: reference.sql
+---
+
+{% include_cached new-in.html version="v23.1" %} `CREATE SCHEDULE FOR CHANGEFEED` allows you to create a scheduled changefeed to export data out of CockroachDB. Scheduled changefeeds have the scale, observability, and endpoint sink options that changefeed jobs include, with the convenience of setting a regular schedule. A changefeed job created with `CREATE SCHEDULE FOR CHANGEFEED` performs a one-time table scan using the [initial scan](create-changefeed.html#initial-scan) functionality to create an export of your table data.
+
+For more detail on using changefeeds to create an export of your table data, see [Export Data with Changefeeds](export-data-with-changefeeds.html).
+
+## Required privileges
+
+{% include {{ page.version.version }}/cdc/privilege-model.md %}
+
+### Legacy privilege model
+
+To create a changefeed, the user must be a member of the `admin` role or have the [`CREATECHANGEFEED`](create-user.html#create-a-user-that-can-control-changefeeds) parameter set.
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/create_schedule_for_changefeed.html %}
+</div>
+
+### Parameters
+
+Parameter | Description
+----------+------------
+`IF NOT EXISTS` | A scheduled changefeed should not be created if the `schedule_label` already exists. You will receive an error if the schedule label already exists, or if `schedule_label` is not defined when using `IF NOT EXISTS`.
+`schedule_label` | The name for the scheduled changefeed. This is optional and does not need to be unique. If you do not define a name, the label will default to `CHANGEFEED` with the timestamp of when you created the schedule. 
+`changefeed_targets` | The tables to target with the changefeed. For example, `movr.users, movr.rides`.
+`changefeed_sink` | A URI to authenticate to the [changefeed sink](changefeed-sinks.html).
```

Comment:
The description can just be "changefeed sink URI", with the hyperlink.

Reply:
Edited!